lib/project.js

```diff
@@ -3,6 +3,9 @@ var git = require('gift');
 var normalizePackageData = require('normalize-package-data');
 var path = require('path');
 
+var BLIP_SCRIPT = 'node scripts/sl-blip.js';
```

Comment:
Having a folder that is empty other than a single dot-file is kind of weird, though, don't you think?